lua/ruby_nvim/cmd.lua

```diff
@@ -85,24 +84,20 @@ end
 
 -- :RubyBrowseGem
 M.browse_gem = function()
-  local line = api.nvim_get_current_line()
-  local gem_index = line:find('gem')
-  local gem_with_space_and_quote_length = 5
-  local quote_and_comma_length = 2
-  local comma_index = line:find(',')
-  local gem_name = ''
-  if not gem_index then
-    vim.notify(string.format("There is no gem in this line"), "error")
+  local node = util.get_method_relevant_to_cursor()
+  if node == nil then
+    vim.notify("gem not found", "error")
```

Comment:
I think the message here might be misleading. The _gem not found_ looks like, to me, that the plugin has got a name for the Gem, but couldn't find it, e.g. in rubygems.org. What about something like _There is no gem close to the cursor_ to keep it aligned with the other error message?

Reply:
Ok, I'll fix it